bugsnag-plugin-android-ndk/src/main/assets/include/event.h

```diff
@@ -48,6 +48,12 @@ typedef enum {
    *  User event, such as authentication or control events
    */
   BSG_CRUMB_USER,
+} bsg_breadcrumb_type_t;
+
+typedef struct {
+    char message[33];
```

Comment:
This character limit has now been lifted. Assuming that we still need a fixed size array on the NDK layer, can we increase this to a less restrictive size? 128 perhaps?

Reply:
Added a migration to achieve this.